boards/xtensa/esp32s2_lolin_mini/esp32s2_lolin_mini-pinctrl.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Google, LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <dt-bindings/pinctrl/esp32s2-pinctrl.h>
+#include <zephyr/dt-bindings/pinctrl/esp32s2-gpio-sigmap.h>
+
+&pinctrl {
+
+	uart0_default: uart0_default {
+		group1 {
+			pinmux = <UART0_TX_GPIO16>;
```

Comment:
you can add the `output-high;` attribute to prevent uart to produce garbage before is properly configured.

Reply:
@sylvioalves ping. Do you want me to fix this first?

Reply:
Fixed. PTAL